### PR TITLE
chore: prep v1.5.0

### PR DESCRIPTION
**Pull request by thaddmt**
- [x] briefly describe the changes in this PR
 - [x] write tests for all new functionality
 - [x] run `npm run docs` and commit changes to API.md
 - [x] update CHANGELOG.md with changes under `main` heading before merging

### CODE REVIEW

**Comment by wipfli** [comment]
Don't forget to run `npm i` to update `package-lock.json`...

**Review by sreeramsama — APPROVED** [review]

**Review by wipfli — CHANGES_REQUESTED** [review]
update package lock file

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@
...
 
+## 1.5.0
+
+### Features / Improvements 🚀
+
+- Update suggestions API to support placeId [#84](https://github.com/maplibre/maplibre-gl-geocoder/pull/84)
+
+### Bug fixes 🐛
+
 ## 1.4.2
```
Comment (wipfli) [inline CHANGELOG.md]:
```suggestion
## 1.4.2
```

**Review by wipfli — APPROVED** [review]
Thanks!

**Pull request merged**